### ISSUE TITLE
Accessibility: missing button description on audio message component

### ISSUE THREAD

**Issue by larrykind** [globaleaks/GlobaLeaks#3626]
### What version of GlobaLeaks are you using?

4.13.6

### What browser(s) are you seeing the problem on?

All

### What operating system(s) are you seeing the problem on?

Windows

### Describe the issue

The action description is missing on all buttons of the audio component. 

### Proposed solution

Insert the action performed by each button adding the tag "aria-label"

**Comment by evilaliv3** [comment]
@larrykind Does this still apply?
